core/modules/dblog/dblog.install

```diff
@@ -99,3 +99,11 @@ function dblog_update_1000() {
     db_create_table('watchdog', backdrop_get_schema_unprocessed('dblog', 'watchdog'));
   }
 }
+
+/**
+ * Set the format for dates in the database log.
+ */
+function dblog_update_1001() {
+  $short = system_get_date_formats('short');
```

Comment:
```suggestion
  $short = system_date_format_load('short');
```

It results in the same thing, but getting a single format back from "get_date_format**s**" makes me cringe.